### PR TITLE
Prevent create dup team

### PR DESCRIPTION
**Pull request by davet1985**
Closes https://github.com/appvia/kore-ui/issues/48

### CODE REVIEW

**Comment by gambol99** [comment]
LGTM

File: lib/components/forms/NewTeamForm.js [hunk 1]
```diff
@@ -50,13 +50,22 @@ class NewTeamForm extends React.Component {
...
         }
         try {
-          const team = await apiRequest(null, 'put', `/teams/${canonicalTeamName}`, Team(canonicalTeamName, spec))
-          await apiRequest(null, 'put', `/teams/${canonicalTeamName}/members/${this.props.user.id}`)
-          await this.props.handleTeamCreated(team)
-          const state = copy(this.state)
-          state.submitting = false
-          this.setState(state)
-          message.success('Team created')
+          const checkTeam = await apiRequest(null, 'get', `/teams/${canonicalTeamName}`)
```
Comment (gambol99) [inline lib/components/forms/NewTeamForm.js]: Would `await` handle an error?

Comment (gambol99) [reply lib/components/forms/NewTeamForm.js]: What i mean is .. if in the rare condition you check for the team .. API errors for some reason .. but you take it as a does not exist?

Comment (davet1985) [reply lib/components/forms/NewTeamForm.js]: Strictly speaking, yes. Calling `await apiRequest` with method `get` will always resolve. If the API fails then the next call to create the team would (probably) also fail...

**Pull request merged**